README.md

```diff
@@ -1 +1,38 @@
-# webpack-Demo
\ No newline at end of file
+# Todo list app
+
+> A Javascript to do list web app that keep track of activities.
+
+## Built With
+
+- Major languages: HTML CSS & JavaScript
+- Frameworks: none
+- Technologies used: Webpack
+
+## Live Demo (if available)
+
```

Comment:
- [ ] Please it will be nice to add explicit steps to clone and run your app locally as this will be of help to non technical users.